src/backend.js

```diff
@@ -290,7 +290,7 @@ export default class Backend extends EventTarget {
 			url: base ? new URL(source, base) : new URL(source)
 		};
 
-		Object.assign(ret, matchURLs(source, this.constructor.urls) ?? matchURLs(source, this.constructor.urlsRecognized))
+		Object.assign(ret, matchURLs(source, this.urls) ?? matchURLs(source, this.urlsRecognized));
```

Comment:
This reminds me, I don't like `urlsRecognized` very much as a name, any ideas what else we could use?

Reply:
Could you elaborate on why we need one more static property for URLs? If I understand it correctly, the URLs you provide inside `urlsRecognized` are URL patterns, too. What am I missing?

Reply:
`urls` are those for which `test()` would return `true`, `urlsRecognized` are URLs that would be *parsed*, but would not affect what `test()` returns. I.e. URLs you could use when using the constructor directly, but wouldn't be used to determine the backend type when you use `Backend.from()`.

Reply:
E.g. in a backend for GitHub labels, you'd only want label URLs to be part of `urls`, but you can still *recognize* repo URLs if the backend is called directly (whereas these URLs would construct a `GithubFile` object if passed into `Backend.from()`).

Reply:
Oh. I see makes sense. Thank you.
Some name ideas:

- `urlsKnown` or `knownUrls`
- `urlsAccepted`
- `urlsOwn` or `ownUrls`

Reply:
I think it should start with `urls` even if it's a little awkward.
I like `urlsKnown` just because it's shorter, but not a huge fan of any of them. A few other options, which I'm also not crazy about:
- `urlsAll`
- `urlsSecondary`
- `urlsAuxiliary`

Reply:
`urlsSupported` as another (brain-stormed) option

Reply:
I'd rather avoid the double `s`, that's why I ruled out `urlsSecondary` too. Let’s use `urlsKnown` for now, as it's the shortest.